project/Dependencies.scala

```diff
@@ -7,9 +7,10 @@ object Dependencies {
   final val scalacheckVersion = "1.14.0"
 
   val scalatestVersion = Def.map(scalaBinaryVersion) {
-    case "2.13.0-M3" => "3.0.5-M1"
-    case "2.13.0-M5" => "3.0.6-SNAP6"
-    case other       => "3.0.5"
+    case "2.13.0-M3"  => "3.0.5-M1"
+    case "2.13.0-M5"  => "3.0.6-SNAP6"
+    case "2.13.0-RC1" => "3.0.8-RC2"
```

Comment:
I notice that Cats depends on `3.1.0-SNAP9` so I assume it's safe enough to use ? I cannot find details / notes about the releases though.

Reply:
Yes, I've used that on other projects.  Since log4s doesn't export a testkit, it doesn't really matter as long as the tests pass, but it would be nice to line them all up on a production version once that's possible.